packages/scss/src/components/inputs/textfield/_textfield.compact.scss

```diff
@@ -42,6 +18,7 @@
 		display: inline-flex;
 		left: calc(100% + 1rem);
```

Comment:
Not related to your update, but as we are refactoring this, we might prefer to use left: 100% and margin-left: 1rem ?

Reply:
✔️